### PR TITLE
added badges

### PR DESCRIPTION
**Pull request by sharmivijay**
kindly review the badges

### CODE REVIEW

File: styles/badges.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    position: relative;
+    font-size: 2rem;
+  }
+  
+  .badge-notification .mail-icon:after {
+    content: "10";
+    background-color: black;
+    position: absolute;
+    top: 0.2rem;
+    right: 0.1rem;
```
Comment (aatishpandey) [inline styles/badges.css:13]: You can use pixels here for sizes less than 10 px for better readability.

File: styles/badges.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    height: 1rem;
+    font-size: 1rem;
+    padding: 0.3rem;
+    border-radius: 1rem;
```
Comment (aatishpandey) [inline styles/badges.css:17]: same here

**Review by aatishpandey — COMMENTED** [review]
The rest looks good. have left a suggestion.

File: index.html [hunk 3]
```diff
@@ -56,6 +61,23 @@ <h4>Different types of alerts we came across in day today life</h4>
...
+        </div>
+
+        <div class="badge-notification">
+            <i class="fa fa-shopping-cart mail-icon"></i>
+        </div>
+        
+        <h4>Badge on Avatar</h4>
 
+        <div class="badge-notification">
+            <img src="images/default-avatar.svg" alt="default-Avatar" class="avatar-large">
```
Comment (krishanu2911) [inline index.html:78]: just a suggestion not mandatory that avatar-large could be avatar-lg a shorter class name try to give smaller class name would be better

File: styles/badges.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    color: white;
+  }
+
+  .badge-notification .avatar-large{
```
Comment (krishanu2911) [inline styles/badges.css:34]: same here

**Review by krishanu2911 — COMMENTED** [review]
Well written code just added a few suggestions else everything is good to go!

**Pull request merged**